Source/WebCore/inspector/InspectorShaderProgram.cpp

```diff
@@ -124,9 +124,19 @@ bool InspectorShaderProgram::updateShader(Inspector::Protocol::Canvas::ShaderTyp
                         contextWebGLBase.shaderSource(*shader, source);
                         contextWebGLBase.compileShader(*shader);
                         auto compileStatus = contextWebGLBase.getShaderParameter(*shader, GraphicsContextGL::COMPILE_STATUS);
-                        if (std::holds_alternative<bool>(compileStatus) && std::get<bool>(compileStatus))
-                            contextWebGLBase.linkProgramWithoutInvalidatingAttribLocations(&program);
-                        return true;
+                        if (std::holds_alternative<bool>(compileStatus)) {
+                            if (std::get<bool>(compileStatus))
+                                contextWebGLBase.linkProgramWithoutInvalidatingAttribLocations(&program);
+                            else {
+                                auto errors = contextWebGLBase.getShaderInfoLog(*shader);
+                                auto* scriptContext = m_canvas.scriptExecutionContext();
+                                for (auto error : StringView(errors).split('\n')) {
+                                    auto message = makeString("Error compiling edited WebGL shader: "_s, error);
```

Comment:
Can we have this just be
```
makeString("WebGL: "_s, error);
```
so that it matches the old error string (and therefore could make the job of identifying WebGL errors easier on the frontend)?